pkg/service/service_test.go

```diff
@@ -1469,23 +1468,34 @@ func (m *ManagerTestSuite) TestL7LoadBalancerServiceOverride(c *C) {
 	// registering without redirecting
 	echoOtherNode := lb.ServiceName{Name: "echo-other-node", Namespace: "cilium-test"}
 	resource1 := lb.ServiceName{Name: "testOwner1", Namespace: "cilium-test"}
-	err = m.svc.RegisterL7LBServiceBackendSync(echoOtherNode, resource1, nil)
+	err = m.svc.RegisterL7LBService(echoOtherNode, resource1, nil, 0)
+	c.Assert(err, IsNil)
+
+	svc, ok = m.svc.svcByID[id]
+	c.Assert(len(svc.backends), Equals, len(allBackends))
+	c.Assert(ok, Equals, true)
+	c.Assert(svc.l7LBProxyPort, Equals, uint16(0))
+
+	// registering backend sync for resource1
+	err = m.svc.RegisterL7LBServiceBackendSync(echoOtherNode, resource1, []string{"8080", "9090", "10000"})
 	c.Assert(err, IsNil)
 
 	svc, ok = m.svc.svcByID[id]
 	c.Assert(len(svc.backends), Equals, len(allBackends))
 	c.Assert(ok, Equals, true)
 	c.Assert(svc.l7LBProxyPort, Equals, uint16(0))
+	c.Assert(svc.l7LBFrontendPorts, IsNil) // No forward active
 
 	// registering with redirection stores the proxy port
 	resource2 := lb.ServiceName{Name: "testOwner2", Namespace: "cilium-test"}
-	err = m.svc.RegisterL7LBService(echoOtherNode, resource2, nil, uint16(9090))
+	err = m.svc.RegisterL7LBService(echoOtherNode, resource2, []string{"8080", "9090", "11000"}, uint16(9090))
```

Comment:
I would suggest adding a comment here on the configuration being tested. Something along the lines of what's mentioned in the commit message "when multiple `CiliumEnvoyConfig`s reference the same
backend service on different ports"

Reply:
good idea - i expanded the comment with a little bit more context.